styles.css

```diff
@@ -22,19 +55,59 @@ body {
     display: none;
   }
 
-  body {
-    background-color: #e5e5e5;
-    margin: 0;
-    padding: 0;
+  .desktop-popup {
+    position: fixed;
+    top: 0;
+    margin: 5%;
+    background-color: #fff;
+    z-index: 5;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    width: 90%;
+    height: 100vh;
+    border-radius: 16px;
+    padding: 2%;
+    backdrop-filter: blur(10px);
   }
 
-  img {
-    transition: transform 1s ease-in-out;
-    transition: all 0.3s;
+  .desktop-popup-img {
+    width: 100%;
+    height: 50%;
   }
 
-  img:hover {
-    transform: scale(1.1);
+  .popup-content-box {
+    display: grid;
+    grid-template-columns: 2fr 1fr;
+  }
```

Comment:
- Great job for implementing popups 👍👍, I think You are losing some of the information from the popup detail because your popup is not scrollable, please make your popup scrollable to view all information 👍👍
- Kindly, the background behind the modal in the Figma design is blurred 👍👍 please make yours also fit the Figma design and have a better UI👍👍


| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img src="https://user-images.githubusercontent.com/52010789/218041171-22eeb59b-4c0d-44cd-b313-487b08becab8.png" width="600">|<img src="https://user-images.githubusercontent.com/52010789/218041152-bc331d9b-8057-440b-aad1-b55d416df4f9.png" width="600">|


😎️

🤝️